packages/reports/src/Stores/Modules/Profit/profit-store.js

```diff
@@ -67,8 +67,14 @@ export default class ProfitTableStore extends BaseStore {
         if (!this.shouldFetchNextBatch(isMounting)) return;
         this.is_loading = true;
         const dateParams = getDateBoundaries(this.date_from, this.date_to, 0, false);
+        const dateParamsDTraderV2 = getDateBoundaries(
```

Comment:
When onMount is from profit-store is called, it reset date filters. In order not to lose them, now I'm taking them from positions store